react-app/src/components/projects/AgencyResponseSearchTable.tsx

```diff
@@ -121,6 +137,7 @@ const AgencySearchTable = (props: IAgencySearchTable) => {
             }}
             component="li"
             {...props}
+            key={option.value}
```

Comment:
I get conflicting ids for keys when opening this autocomplete. Might have to be more specific for the key.